test/sqatt/src/Sqatt.hs

```diff
@@ -154,6 +159,13 @@ decodePath filePath =
       throwError $ FilePathError $
         "Cannot decode " <> apprPath <> " properly"
 
+-- | Replace the characters that might cause problems on Windows systems.
```

Comment:
Why are problems limited to Windows systems?
Why not the more general `Replace characters that could cause problems on any platform`?
Does this include Unicode?

Reply:
I only run into issues on Windows so far. If we run into issues with an Unix based OS we can rename the function accordingly. This function is not related to Unicode handling but to replacing those characters that cannot be used on Windows (`:`).